Tests/CSharp/ExpressionTests/ExpressionTests.cs

```diff
@@ -1061,7 +1061,7 @@ private partial class Rec
 
     private Rec TestMethod(string str)
     {
-        int length = str?.Length ?? -1;
+        int length = (str?.Length) ?? -1;
```

Comment:
New Roslyn doesn't perform some simplifications?

Reply:
Perhaps the default options for simplification changed and this needs explicitly enabling in some way now?
I noticed that one of the options previously used is deprecated too.

Reply:
I wonder if some of the simplifications were abandoned when trying to make them handle the "target-typed new", and "target-typed conditional expressions"

Reply:
I noticed in the renamer that the logic for locating symbols from a different compilation has got worse, I wonder if that's influencing any of the code around simplification